frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'http://localhost:5143'
+  apiUrl: 'http://localhost:5000'
```

Comment:
imo we should add dev environments in .gitignore and keep file structure in production versions of those files, because, for example, i cannot run app on 5000 port

Reply:
@goose0chka If you really wish, we can use port 6000, just it should be changed everywhere

Reply:
Don't worry, I keep my settings in stash